README.md

```
@@ -201,6 +202,16 @@ Continue by accepting the list in the web browser dashboard. Then, in the django
 ./scripts/manage batch_process -a match -l 16
 ```
 
+### Process partner data file uploads
+
+Moderators submit partner Google Sheets via Django admin (**Partner Data File Uploads**); deployed environments run this through AWS Batch. Locally, create an upload pointing at a shared Google Sheet (status `PROCESSING`), then run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)